### PR TITLE
GHA - new actions to add/remove wating-response

### PR DESCRIPTION
**Pull request by catriona-m**
This should:
- add a label for "waiting-response" when a PR has been reviewed but not yet approved
- remove "waiting-response" label when new commits are made to a PR.

### CODE REVIEW

File: .github/workflows/pull-request-new-commit.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  pull_request:
+    types: [synchronize]
+
+jobs:
+  issue_comment_triage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions-ecosystem/action-remove-labels@v1
+        with:
+          github_token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (tombuildsstuff) [inline .github/workflows/pull-request-new-commit.yaml:14]: does this need permissions? by default the `GITHUB_TOKEN` returned only includes Read rules

Comment (catriona-m) [reply .github/workflows/pull-request-new-commit.yaml:14]: Not that I can see, I tested this in my own repo and it seems to work. Also used this as a reference and it doesn't seem to need permissions either? https://github.com/hashicorp/terraform-provider-azurerm/blob/main/.github/workflows/pull-request.yaml

**Review by katbyte — COMMENTED** [review]
Is it possible to limit which users trigger removing the label? Often when i add it after i leave a comment, its removed

image

**Comment by catriona-m** [comment]
> Is it possible to limit which users trigger removing the label? Often when i add it after i leave a comment, its removed
> 
> image

I've added a change in issue-comment-created.yaml now that only removes waiting-response from a PR comment if it's the author of the PR that has replied. If you think that would work here?

**Comment by katbyte** [comment]
Thats perfect! thanks

**Review by katbyte — APPROVED** [review]
LGTM 🎩

**Pull request merged**